From 3b5cd332ab874e93c2474aa24df4013328128523 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Fri, 3 Jan 2025 14:51:28 +0100
Subject: [PATCH 1426/1695] plat-stm32mp2: increase heap to 384kB

Memory heap requirement when adding pm callbacks to fixed regulator :
plat-stm32mp2: stm32mp25_pwr: restore fixed regulator

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 478d80a566fbe48e80da383c8b8a775fc801178e plat-stm32mp2: increase heap to 384kB]
Change-Id: I0c08a5d43d3ee5a846e0f1d6e554e25379433acf
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/426918
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 3c186ab9d..4ae2b7897 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -143,7 +143,7 @@ CFG_TZDRAM_SIZE  ?= 0x02000000
 CFG_CORE_LARGE_PHYS_ADDR ?= y
 CFG_CORE_ARM64_PA_BITS ?= 34
 
-CFG_CORE_HEAP_SIZE ?= 262144
+CFG_CORE_HEAP_SIZE ?= 393216
 CFG_CORE_RESERVED_SHM ?= n
 CFG_DTB_MAX_SIZE ?= 262144
 CFG_HALT_CORES_ON_PANIC ?= y
-- 
2.47.2

